rfcs/0153-uefi-only.md

```diff
@@ -0,0 +1,174 @@
+---
+feature: non_legacy_boot
+start-date: 2023-06-16
+author: Ryan Lahfa
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+NixOS will have first-class support for UEFI
+and uses it as a default boot environment, for supported architectures,
```

Comment:
```suggestion
and uses it as a default boot environment, on supported machines,
```

UEFI does not "support" entire architectures in any meaningful sense.  It "supports" MIPS and yet there is not a single UEFI MIPS machine in existence!  Talking about "UEFI-supported architectures" is extremely misleading.